### PR TITLE
Updated units in locations

### PR DESCRIPTION
**Pull request by AnAngrySalad1**
- Significantly upped numbers of ai in locations, too often do we get into a town and capture it having engaged all the enemies on the outside.

### CODE REVIEW

File: addons/occupy/functions/fnc_setOccupied.sqf [hunk 1]
```diff
@@ -125,7 +125,7 @@ _icon setMarkerText (["Liberate",_typeName] joinString " ");
 _icon setMarkerType "o_installation";
 _mrkArray pushBack _icon;
 
-GVAR(location) = [_name,_center,_size,_type]; // set as occupied location
+GVAR(location) = [_name,_center,_,_type]; // set as occupied location
```
Comment (tbeswick96) [inline addons/occupy/functions/fnc_setOccupied.sqf:128]: `_` looks like an error

**Review by tbeswick96 — CHANGES_REQUESTED** [review]

File: addons/occupy/functions/fnc_setOccupied.sqf (hunk 1 above)
Comment (AnAngrySalad1) [reply addons/occupy/functions/fnc_setOccupied.sqf:128]: I dont think i changed that? Or is this something you've spotted

Comment (tbeswick96) [reply addons/occupy/functions/fnc_setOccupied.sqf:128]: It's in the pull request as a change, probably a misclick or something

Comment (AnAngrySalad1) [reply addons/occupy/functions/fnc_setOccupied.sqf:128]: ok then my bad

**Comment by tbeswick96** [comment]
Resolved in files directly

**Pull request closed without merging**